msk/actions/create.py

```diff
@@ -257,6 +257,41 @@ def commit_changes(self):
             self.git.add('.')
             self.git.commit(message='Initial commit')
 
+    def force_push(self, get_repo_name: Callable = None) -> Optional[Repository]:
+        if ask_yes_no(
+                'Are you sure you want to overwrite the remote github repo? '
+                'This cannot be undone and you will lose your commit '
+                'history! (y/N)',
+                False):
+            repo_name = (get_repo_name and get_repo_name()) or (
+                    self.name + '-skill')
+            repo = self.user.get_repo(repo_name)
+            call(['git', 'push', '-f', 'origin', 'master'],
+                 cwd=self.git.working_dir)
+            print('Force pushed to GitHub repo:', repo.html_url)
+            return repo
+
+    def link_github_repo(self, get_repo_name: Callable = None) -> Optional[Repository]:
+        if 'origin' not in Git(self.path).remote().split('\n'):
+            if ask_yes_no(
+                    'Would you like to link an existing GitHub repo to it? (Y/n)',
+                    True):
+                repo_name = (get_repo_name and get_repo_name()) or (
+                        self.name + '-skill')
+                repo = self.user.get_repo(repo_name)
+                self.git.remote('add', 'origin', repo.html_url)
+                self.git.fetch()
+                try:
+                    self.git.pull('origin', 'master')
+                except GitCommandError as e:
+                    if e.status == 128:
+                        raise UnrelatedGithubHistory(repo_name) from e
+                    raise
+                call(['git', 'push', '-u', 'origin', 'master'],
```

Comment:
Like above, this can also be: `self.git.push('origin', 'master', set_upstream=True)`